format.go

```diff
@@ -93,9 +94,11 @@ func toPprof(s map[string]int, hz int) *profile.Profile {
 }
 
 func sortedKeys(s map[string]int) []string {
-	var keys []string
+	keys := make([]string, len(s))
+	i := 0
 	for stack := range s {
-		keys = append(keys, stack)
+		keys[i] = stack
+		i++
```

Comment:
Similar to https://github.com/felixge/fgprof/pull/7, this change is a bit of a premature optimization since it doesn't fall in the critical path of the sampling itself. That being said, I appreciate your PRs and the comment improvements you're also making along the way, so I'll go ahead and merge this as well.

That being said, if you want to make more performance related PRs, try tackling stuff in the critical sampling path. Basically anything that's invoked from this for loop here:

https://github.com/felixge/fgprof/blob/master/fgprof.go#L29-L37